warehouse/packaging/utils.py

```diff
@@ -22,7 +22,7 @@
 from warehouse.packaging.interfaces import ISimpleStorage
 from warehouse.packaging.models import File, LifecycleStatus, Project, Release
 
-API_VERSION = "1.2"
+API_VERSION = "1.3"
```

Comment:
This comes from the PEP itself: 

> The api-version SHALL specify version 1.3 or later.

(Via <https://peps.python.org/pep-0740/#json-based-simple-api>)

In this context, all incrementing it does is signal to a client that the index is now serving PEP 740 objects. Clients are expected to handle this gracefully, since `api-version` is intended to be a semantic version.

(I can find a cite for that last claim if you'd like, although I'd have to go digging. But for context, this was just bumped a few weeks ago from `1.1` to `1.2` with no client breakage due to PEP 708.)